app/models/pregnancy.rb

```diff
@@ -7,9 +7,9 @@ class Case
 
 	field :line, type: String #DC, MD, VA
 	field :language, type: String
-	field :case_id, type: String
+	field :p_case_id, type: String
```

Comment:
@Kevin-Wei let's switch it to `pregnancy_id` and `pregnancy_status`, since I don't think there's another instance of `p_{whatever}` anywhere at the moment.